### DISCUSSION TITLE
Domain future

### DISCUSSION THREAD

**Discussion by darky** [zerobias/effector#1175] [General]
Hello! I have a question about Domain future. There is suspicion, that it probably will be deprecated in future releases of Effector, some red bells:

* [You Don't Need Domains](https://withease.effector.dev/magazine/no_domains.html)
*  ̶[f̶o̶r̶k̶(̶d̶o̶m̶a̶i̶n̶,̶ ̶o̶p̶t̶i̶o̶n̶s̶?̶)̶ deprecation](https://effector.dev/en/api/effector/fork/#methods-fork-domain)

Is I properly understand, that better don't touch Domain at all?

**Comment by sergeysova** [comment]
Hello!
Thank you for the question.

At the moment domains are not deprecated, but will be in future releases

**Reply by darky** [reply]
Thanks for answer!
